isotope/service/pkg/srv/handler.go

```diff
@@ -44,15 +44,15 @@ func (h Handler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
 
 	respond := func(status int) {
 		writer.WriteHeader(status)
-		err := request.Write(writer)
-		if err != nil {
+		payloadSize := uint64(h.Service.ResponseSize)
+		payload := make([]byte, payloadSize)
```

Comment:
Given that `payloadSize` never changes during the service's lifetime, it would make sense to make `payload` global. My logic here is that this better simulates a real service's response payload, but I'd be interested in thoughts.

Reply:
Okay. Done.